history.go

```diff
@@ -65,3 +65,11 @@ func (h *History) NextEpoch() int64 {
 func (h *History) MerkleTree() *MerkleTree {
 	return h.tree
 }
+
+func (h *History) NewTemporaryBinding(index, value []byte) []byte {
```

Comment:
I'm thinking it might be better to have a separate TB struct, because the server needs to keep track of all temporary bindings issued during a given epoch, and return a TB whenever a client makes a lookup for a name which may not be in the tree yet but which has been registered/changed. 

I think the functionality of keeping track can probably be implemented in the server itself, but I think it would facilitate handling TBs if they had their own struct.